dependencies.props

```diff
@@ -133,6 +134,17 @@
       <ElementName>MicrosoftNETCoreRuntimeCoreCLRPackageVersion</ElementName>
       <PackageId>Microsoft.NETCore.Runtime.CoreCLR</PackageId>
     </XmlUpdateStep>
+    <XmlUpdateStep Include="CoreClr">
+      <Path>$(MSBuildThisFileFullPath)</Path>
+      <ElementName>MicrosoftNetCoreIlasmPackageVersion</ElementName>
+      <PackageId>Microsoft.NETCore.ILAsm</PackageId>
+    </XmlUpdateStep>
+    <UpdateStep Include="CoreCLR">
+      <UpdaterType>MSBuildSdk</UpdaterType>
```

Comment:
Nice I didn't realize we already had this updater.

Reply:
We didn't. 😄  @tannergooding added it.  https://github.com/dotnet/buildtools/pull/2117

Welcome back, BTW.

Reply:
Great thanks @tannergooding for adding this support.